### PR TITLE
Automate build check and installation check

### LINKED ISSUE #16: CI workflow for checking if the node can be installed
<a href="https://107-systems.org/"></a>

### ⚡ Feature Request

**Please Describe The Problem To Be Solved**

Create a CI workflow that checks, if the node can be installed correctly.

### PR DESCRIPTION
**Pull request by wuehr1999**
Fixes #16

@aentinger I created an action, which checks if the service definitions of the node can be built and if the python wheel can be installed without problems.

### CODE REVIEW

File: .github/workflows/ros2.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        source /opt/ros/${{ matrix.ros_distro }}/setup.bash
+        mkdir -p ${{github.workspace}}/ros2_ws/src
+        cd ${{github.workspace}}/ros2_ws
+        colcon build
+        
+    - uses: actions/checkout@v2
+      with:
+        repository: '107-systems/l3xz_openmv_camera'
+        path: 'ros2_ws/src/l3xz_openmv_camera'
+
```
Comment (per1234) [inline .github/workflows/ros2.yml:34]:
```suggestion
```
Unless I miss something, this was redundant to the next step.

File: .github/workflows/ros2.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        mkdir -p ${{github.workspace}}/ros2_ws/src
+        cd ${{github.workspace}}/ros2_ws
+        colcon build
+        
+    - uses: actions/checkout@v2
+      with:
+        repository: '107-systems/l3xz_openmv_camera'
+        path: 'ros2_ws/src/l3xz_openmv_camera'
+        
+    - uses: actions/checkout@v2
```
Comment (per1234) [inline .github/workflows/ros2.yml:35]:
```suggestion
    - uses: actions/checkout@v3
```
Unless there is a specific reason for using the older action version, it is best to start with the up to date version.

https://github.com/actions/checkout/releases/tag/v3

**Review by per1234 — DISMISSED** [review]

File: .github/workflows/ros2.yml (hunk 2 above)
Comment (wuehr1999) [reply .github/workflows/ros2.yml]: @per1234 thanks for Your hints, I missed those points :+1:

**Review by aentinger — APPROVED** [review]
LGTM :+1: Thank you @wuehr1999 :rocket:

**Pull request merged**